documentation/modules/ROOT/pages/operations/debezium-platform.adoc

```diff
@@ -359,25 +383,46 @@ For example, consider the following JSON for configuring a sink destination in {
 debezium.sink.type=pubsub
 debezium.sink.pubsub.project.id=debezium-tutorial-local
 debezium.sink.pubsub.address=pubsub:8085
+debezium.source.connector.class=io.debezium.connector.mysql.MySqlConnector
+debezium.source.database.hostname=mysql
+debezium.source.database.server.id=223344
+debezium.source.database.port=3306
+debezium.source.database.user=debezium
+debezium.source.database.password=dbz
+debezium.source.schema.history.internal=io.debezium.storage.file.history.FileSchemaHistory
+debezium.source.schema.history.internal.file.filename=data/schema.dat
+debezium.source.offset.storage.file.filename=data/offsets.dat
+debezium.source.offset.flush.interval.ms=0
+debezium.source.topic.prefix=tutorial
+debezium.source.database.include.list=inventory
+debezium.source.table.include.list=inventory.customers
 
 # ..
 ----
 
-To adapt the preceding configuration for use in defining a sink destination in the **Smart Editor**,  remove the prefix `debezium.sink.pubsub` and convert the configuration to JSON format.
+When you add this properties file to the Smart Editor, it intelligently recognizes that the provided configuration is a Debezium Server configuration and will inform via displaying an alert.
+
+image::debezium-smart-editor-server.png[{prodname} UI smart editor - Server]
 
-The following example shows the JSON that you might use to define a sink destination in the **Smart Editor**.
+The Smart Editor toolbar will then display an **Auto Format** option (highlighted in the image above). You can use this option to automatically extract the sink configuration and convert the sink properties into the required Debezium Platform JSON format.
+
+The following image shows the auto formatted JSON that you would use to define a Pub-Sub sink where you have to just provide name and optionally add some appropriate description in the JSON.
```

Comment:
```suggestion
The following image shows automatically formatted JSON that defines a publish-subscribe sink.
Edit the JSON to update the value of the `name` field, and optionally add populate the `description` field.
```

Reply:
feels like something is missing here, "_and optionally add populate the `description` field._" ?